test/smoke/src/sql/queryEditor/queryEditor.test.ts

```diff
@@ -4,12 +4,22 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Application } from '../../application';
+import { allExtensionsLoadedText } from '../../main';
 import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
 
 export function setup() {
 	describe('Query Editor Test Suite', () => {
+		before(async function () {
```

Comment:
I'm not sure why this would change anything. This  is already done here https://github.com/microsoft/azuredatastudio/blob/dce115fb82ec64d3629ef78ea3da01ffd030a830/test/smoke/src/main.ts#L256.

Reply:
Well shoot, yeah I saw that but for some reason thought we were doing a reload of the app after waiting. The issue is definitely because focus is being taken from the query window though - although why that's happening after the service should be done downloading I'm not sure...it doesn't repro consistently enough to really help here.

I'll try changing it to just specifically focus the query pane before typing the text - that should also solve the problem unless the focus stealing is happening even after the download is finished.

Reply:
it this is being caused because of the service downloading, you could change the output channel logic in the extensions to not continuously open the output log.  https://github.com/microsoft/azuredatastudio/blob/master/extensions/mssql/src/main.ts#L365 I have been thinking about doing this, but its been low pri since it has no user impact on our releases since we never download new service binaries for releases.